File: 백준/Bronze/5533. 유니크/유니크.c
#include <stdio.h>
#include <stdlib.h>
int main(void) {
    int n, total, cnt, *arr, **score;
    scanf("%d", &n);
    arr = (int*)malloc(sizeof(int) * 3);
    score = (int**)malloc(sizeof(int*) * 3);
    for (int i = 0; i < 3; i++)
        score[i] = (int*)malloc(sizeof(int) * n);
    for (int i = 0; i < n; i++) {
        for (int j = 0; j < 3; j++) {
            scanf("%d", &arr[j]);
            score[j][i] = arr[j];
        }
    }
    for (int i = 0; i < n; i++) {
        total = 0;
        for (int j = 0; j < 3; j++) {
            cnt = 0;
            for (int k = 0; k < n; k++) {
                if (score[j][i] == score[j][k])
                    cnt += 1;
            }
            if (cnt == 1) {
                total += score[j][i];
            }
        }
        printf("%d\n", total);
    }
    return 0;
}

File: 백준/Bronze/5533. 유니크/유니크.py
import sys
input=lambda:sys.stdin.readline().rstrip()
n=int(input())
score=[[],[],[]]
for _ in range(n):
    arr=[*map(int,input().split())]
    for j in range(3):
        score[j].append(arr[j])
for i in range(n):
    total=0
    for j in range(3):
        if score[j].count(score[j][i])==1:
            total+=score[j][i]
    print(total)